phpunit/block-supports/layout-test.php

```diff
@@ -7,6 +7,21 @@
  */
 
 class WP_Block_Supports_Layout_Test extends WP_UnitTestCase {
+	/**
+	 * @var string|null
+	 */
+	private $theme_root;
+
+	/**
+	 * @var array|null
+	 */
+	private $orig_theme_dir;
+
+	/**
+	 * @var array|null
+	 */
+	private $queries;
+
```

Comment:
are these (and the similar ones below) included for PHP compatibility issues?

Reply:
Yes. These properties were added to ensure compatibility with PHP 8.2 and newer PHP versions.
PHP 8.2 deprecates dynamic class properties.